tests/assets/ansible/roles/caasp/tasks/Suse.yaml

```diff
@@ -1,6 +1,8 @@
 ---
 - name: add repositories
   vars:
+    sle_major: "{{ ansible_distribution_version.split('.')[0] }}"
+    sle_minor: "{{ ansible_distribution_version.split('.')[1] }}"
```

Comment:
Not needed (you are not using it)

Reply:
This is only because we switched back to hardcoded values for the time being.
It is better to have all repo deployment code constructed the same way imho.

Reply:
the variables are not used - I don't see why we want to define them then.
and if we want to have them defined the same way globally, I think we would need to do that outside of a single role.

Reply:
I wanted to make it similar to what is done in ansible/roles/node-base.
The thing is that for now we have to hardcode repo for Caasp.

It it much easier to read ansible code in various module when they are all written the same way imho.

I'll remove it though.